introduction.md

```diff
@@ -2,6 +2,10 @@ Dear Dojo community
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?